decentraland/explorer
# fix: rely on "dirty" attribute

How to test this:

* set `DEBUG_SYNC_SYSTEM` to true
* make sure all the logging entries are always good misses or good hits

Start reviewing here: https://github.com/decentraland/explorer/pull/261/files#diff-009b9ffa1533b81274550200ceeba6beL143
